bigml/ensemble.py

```diff
@@ -178,17 +181,40 @@ def __init__(self, ensemble, api=None, max_models=None, cache_get=None):
                         query_string=query_string,
                         no_check_fields=no_check_fields)
 
+                models = [model]
+
+        if self.distributions is None:
+            try:
+                self.distributions = [{'training': {'categories': m.tree.distribution}}
```

Comment:
Usually pylint complains about variables with names shorter than three letters. We usually write down the whole name in the bindings. So this would be 

[{'training': {'categories': model.tree.distribution}} for model in models]